vendor/composer/autoload_static.php

```diff
@@ -0,0 +1,184 @@
+        'iProvider' => __DIR__ . '/../..' . '/symphony/lib/interface/interface.provider.php',
+        'migration_220' => __DIR__ . '/../..' . '/install/migrations/2.2.0.php',
+        'migration_221' => __DIR__ . '/../..' . '/install/migrations/2.2.1.php',
+        'migration_222' => __DIR__ . '/../..' . '/install/migrations/2.2.2.php',
+        'migration_223' => __DIR__ . '/../..' . '/install/migrations/2.2.3.php',
+        'migration_224' => __DIR__ . '/../..' . '/install/migrations/2.2.4.php',
+        'migration_225' => __DIR__ . '/../..' . '/install/migrations/2.2.5.php',
+        'migration_230' => __DIR__ . '/../..' . '/install/migrations/2.3.0.php',
+        'migration_231' => __DIR__ . '/../..' . '/install/migrations/2.3.1.php',
+        'migration_232' => __DIR__ . '/../..' . '/install/migrations/2.3.2.php',
+        'migration_233' => __DIR__ . '/../..' . '/install/migrations/2.3.3.php',
+        'migration_234' => __DIR__ . '/../..' . '/install/migrations/2.3.4.php',
+        'migration_235' => __DIR__ . '/../..' . '/install/migrations/2.3.5.php',
+        'migration_236' => __DIR__ . '/../..' . '/install/migrations/2.3.6.php',
+        'migration_240' => __DIR__ . '/../..' . '/install/migrations/2.4.0.php',
+        'migration_250' => __DIR__ . '/../..' . '/install/migrations/2.5.0.php',
+        'migration_251' => __DIR__ . '/../..' . '/install/migrations/2.5.1.php',
+        'migration_252' => __DIR__ . '/../..' . '/install/migrations/2.5.2.php',
+        'migration_253' => __DIR__ . '/../..' . '/install/migrations/2.5.3.php',
+        'migration_260' => __DIR__ . '/../..' . '/install/migrations/2.6.0.php',
+        'migration_261' => __DIR__ . '/../..' . '/install/migrations/2.6.1.php',
+        'migration_2610' => __DIR__ . '/../..' . '/install/migrations/2.6.10.php',
+        'migration_2611' => __DIR__ . '/../..' . '/install/migrations/2.6.11.php',
+        'migration_262' => __DIR__ . '/../..' . '/install/migrations/2.6.2.php',
+        'migration_263' => __DIR__ . '/../..' . '/install/migrations/2.6.3.php',
+        'migration_264' => __DIR__ . '/../..' . '/install/migrations/2.6.4.php',
+        'migration_265' => __DIR__ . '/../..' . '/install/migrations/2.6.5.php',
+        'migration_266' => __DIR__ . '/../..' . '/install/migrations/2.6.6.php',
+        'migration_267' => __DIR__ . '/../..' . '/install/migrations/2.6.7.php',
+        'migration_268' => __DIR__ . '/../..' . '/install/migrations/2.6.8.php',
+        'migration_269' => __DIR__ . '/../..' . '/install/migrations/2.6.9.php',
+        'migration_270' => __DIR__ . '/../..' . '/install/migrations/2.7.0.php',
+    );
+
+    public static function getInitializer(ClassLoader $loader)
+    {
+        return \Closure::bind(function () use ($loader) {
+            $loader->classMap = ComposerStaticInit5f867b58cbaae676dc2f623c94dcd5ad::$classMap;
+
+        }, null, ClassLoader::class);
+    }
+}
```

Comment:
Do we still need the vendor/composer/autoload_classmap.php file if we have this one instead ?